example.env.js

```diff
@@ -0,0 +1,15 @@
+// GOOGLE_CLIENT_ID=1098968195328-m288ardavvmlsegjnd6iudtecsr8revu.apps.googleusercontent.com
+// GOOGLE_CLIENT_SECRET=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
+// CLOUDINARY_NAME=trailfeathers
+// CLOUDINARY_API_KEY=583278364999251
+// CLOUDINARY_API_SECRET=hcPbaWNlAQLmqEMaMco91Suddss
+// CLOUDINARY_URL=cloudinary://583278364999251:hcPbaWNlAQLmqEMaMco91Suddss@trailfeathers
+// X-RapidApi-Key=2d8ea84d38msh3d15d68f2486f87p1f6cb5jsn33d451b5a65d
+// X-RapidAPI-Host=trailapi-trailapi.p.rapidapi.com
+// RDS_HOSTNAME=feathersdb.caditywwd2mr.us-west-2.rds.amazonaws.com
+// RDS_USERNAME=root
+// RDS_PASSWORD=OpSpark2022!
+// RDS_PORT=3306
+
+// WEATHER=1cba7e8096842a643564f2deaef6e6d1
+// REACT_APP_GOOGLE_MAPS_API_KEY=AIzaSyBXzL3HavgvvGAQci2vEK4LXbq3f3HE3mE 
```

Comment:
kill all of these credentials now that they are available to everyone - never push up sensitive information